src/Program.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using DSharpPlus;
+using DSharpPlus.CommandAll;
+using DSharpPlus.CommandAll.Parsers;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NotifierRedirecter.Events;
+using Serilog;
+using Serilog.Events;
+using Serilog.Sinks.SystemConsole.Themes;
+
+namespace NotifierRedirecter;
+
+public sealed class Program
+{
+    internal static readonly IServiceCollection Services;
+    internal static readonly IConfiguration Configuration;
+    internal static readonly Database Database;
+    internal static readonly ILoggerFactory LoggerFactory;
+    private static readonly ILogger<Program> Logger;
+
+    static Program()
```

Comment:
```suggestion
    static Program()
```
I prefer to be explicit with access modifier

Reply:
Ignore this. I did not know static constructors was a thing and thought this was some weird method

Reply:
Static constructors don't have modifiers.